cmd/vcert/flags.go

```diff
@@ -423,6 +423,15 @@ var (
 		Destination: &flags.omitSans,
 	}
 
+	flagCSRFormat = &cli.StringFlag{
+		Name: "format",
+		Usage: "Generates the Certificate Signing Request in the specified format. Options include: pem | json\n" +
+			"\t\tpem: Generates the CSR in classic pem format to be used as a file.\n" +
+			"\t\tjson: Generates the CSR in json format, suitable for rest api operations.",
```

Comment:
Sound be capitalized JSON, REST, and API in the description.  Also, for some reason the formatting of these tabs looks off when I build/execute on Linux.  Please investigate that, too.

Reply:
Capitalized JSON, REST and API keywords